api/src/main/java/org/openmrs/Encounter.java

```diff
@@ -384,6 +386,21 @@ public Patient getPatient() {
 	public void setPatient(Patient patient) {
 		this.patient = patient;
 	}
+
+	/**
+	 * Gets the set of diagnoses
+	 * @return Returns the set of diagnoses.
+	 */
+	public Set<Diagnosis> getDiagnoses() {
+		return diagnoses;
+	}
+
+	/**
```

Comment:
This is not correct javadoc

Reply:
Yes, I did take a look. I have refactored it accordingly.